app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html

```diff
@@ -27,18 +27,39 @@
     paragraphBody: paragraphBody
 )
 
-@(form: Form[_], onSubmit: Call)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], organisationName: String, mode: Mode)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, "heading")) {
+@input(formName: String, key: String, id: String) = @{
```

Comment:
id is not being used here, Also I think the ID should match the form field name so that the # for the error link works, as at the moment will they all be "value" which is a no-no

Reply:
if I remember properly i InputViewModel already put an ID as the field name and so allow to href the error message to the code. If we put an ID manually we need to make sure this is the same as the field name, otherwise the href will be different and the error message will not focus on the input when clicking